pkg/golinters/gocritic.go

```diff
@@ -92,15 +94,15 @@ func (w *goCriticWrapper) init(settings *config.GoCriticSettings, logger logutil
 	w.once.Do(func() {
 		err := checkers.InitEmbeddedRules()
 		if err != nil {
-			logger.Fatalf("%s: %v: setting an explicit GOROOT can fix this problem.", goCriticName, err)
+			logger.Fatalf("%s: %v: setting an explicit GOROOT can fix this problem", goCriticName, err)
 		}
 	})
 
 	settingsWrapper := newGoCriticSettingsWrapper(settings, logger)
-
-	settingsWrapper.inferEnabledChecks()
-
-	if err := settingsWrapper.validate(); err != nil {
+	settingsWrapper.InferEnabledChecks()
+	// NOTE(Antonboom): Validate must be after InferEnabledChecks, not before.
```

Comment:
```suggestion
	// Validate must be after InferEnabledChecks, not before.
```

Reply:
Why you don't like noted comments? :)
helps to find the author in case of misunderstanding..

Reply:
Notes are just comments, we don't need to know who writes a comment.
This is different for FIXME and TODO.